<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,0">
    <item>
     <layout class="QHBoxLayout" name="sceneLayout"/>
    </item>
    <item>
     <widget class="QWidget" name="preferencesWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>330</width>
            <height>727</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QGroupBox" name="gaussianPreferences">
             <property name="title">
              <string>Gaussian blur settings</string>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="1">
               <widget class="QSlider" name="blurPassingNumberSlider">
                <property name="maximum">
                 <number>5</number>
                </property>
                <property name="singleStep">
                 <number>1</number>
                </property>
                <property name="pageStep">
                 <number>0</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Blur passing number</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QSpinBox" name="blurPassingNumberSBox">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="buttonSymbols">
                 <enum>QAbstractSpinBox::NoButtons</enum>
                </property>
                <property name="suffix">
                 <string/>
                </property>
                <property name="prefix">
                 <string/>
                </property>
                <property name="maximum">
                 <number>5</number>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="tintPreferences">
             <property name="title">
              <string>Tint settings</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <layout class="QVBoxLayout" name="tintColorLayout">
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>Tint color</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>Tint opacity</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSlider" name="tintOpacitySlider">
                  <property name="maximum">
                   <number>1000</number>
                  </property>
                  <property name="singleStep">
                   <number>10</number>
                  </property>
                  <property name="pageStep">
                   <number>0</number>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="tintOpacityDSBox">
                  <property name="minimumSize">
                   <size>
                    <width>100</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>100</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                  <property name="buttonSymbols">
                   <enum>QAbstractSpinBox::NoButtons</enum>
                  </property>
                  <property name="prefix">
                   <string/>
                  </property>
                  <property name="suffix">
                   <string> %</string>
                  </property>
                  <property name="maximum">
                   <double>1.000000000000000</double>
                  </property>
                  <property name="singleStep">
                   <double>0.010000000000000</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QComboBox" name="tintCompsitionModeCBox"/>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="noisePreferences">
             <property name="title">
              <string>Noise settings</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Noise opacity</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="noiseOpacitySlider">
                <property name="maximum">
                 <number>1000</number>
                </property>
                <property name="singleStep">
                 <number>5</number>
                </property>
                <property name="pageStep">
                 <number>0</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="noiseOpacityDSBox">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="buttonSymbols">
                 <enum>QAbstractSpinBox::NoButtons</enum>
                </property>
                <property name="suffix">
                 <string> %</string>
                </property>
                <property name="decimals">
                 <number>3</number>
                </property>
                <property name="maximum">
                 <double>1.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.005000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="windowPreferences">
             <property name="title">
              <string>Window settings</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>Border relative size</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSlider" name="borderRelativeSizeSlider">
                  <property name="maximum">
                   <number>20</number>
                  </property>
                  <property name="singleStep">
                   <number>1</number>
                  </property>
                  <property name="pageStep">
                   <number>0</number>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="borderRelativeSizeSBox">
                  <property name="minimumSize">
                   <size>
                    <width>100</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>100</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                  <property name="buttonSymbols">
                   <enum>QAbstractSpinBox::NoButtons</enum>
                  </property>
                  <property name="suffix">
                   <string> %</string>
                  </property>
                  <property name="maximum">
                   <number>20</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QPushButton" name="btn_background1">
                  <property name="minimumSize">
                   <size>
                    <width>90</width>
                    <height>50</height>
                   </size>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="resources.qrc">
                    <normaloff>:/Content/Textures/background.png</normaloff>:/Content/Textures/background.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>80</width>
                    <height>45</height>
                   </size>
                  </property>
                  <property name="checkable">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="btn_background2">
                  <property name="minimumSize">
                   <size>
                    <width>90</width>
                    <height>50</height>
                   </size>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="resources.qrc">
                    <normaloff>:/Content/Textures/background2.png</normaloff>:/Content/Textures/background2.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>80</width>
                    <height>45</height>
                   </size>
                  </property>
                  <property name="checkable">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="btn_background3">
                  <property name="minimumSize">
                   <size>
                    <width>90</width>
                    <height>50</height>
                   </size>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="resources.qrc">
                    <normaloff>:/Content/Textures/background3.png</normaloff>:/Content/Textures/background3.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>80</width>
                    <height>45</height>
                   </size>
                  </property>
                  <property name="checkable">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>350</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1366</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
